src/Platform/TargetPlatform.php

```diff
@@ -28,6 +29,7 @@ public function __construct(
         public readonly ThreadSafetyMode $threadSafety,
         public readonly int $makeParallelJobs,
         public readonly WindowsCompiler|null $windowsCompiler,
+        public readonly PhpizePath $phpizePath,
```

Comment:
Not sure about this change overall currently; we can't necessarily eagerly try to determine the `phpize` path; since Windows doesn't have phpize in the same way; so this will completely break on Windows.

Reply:
Indeed all Windows tests are broken right now. I'll try to figure out another solution